app/src/main/java/com/locoquest/app/dto/User.kt

```diff
@@ -1,5 +1,3 @@
 package com.locoquest.app.dto
 
-data class User(
-    val uid: String,
-    val displayName: String)
+data class User(val uid: String, val displayName: String)
```

Comment:
Since there are only a few primary constructor parameters, you can reduce this to one line

https://kotlinlang.org/docs/coding-conventions.html#class-headers

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.